staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -462,27 +471,32 @@ func (s *store) GuaranteedUpdate(
 			clientv3.OpGet(key),
 		).Commit()
 		metrics.RecordEtcdRequestLatency("update", s.groupResourceString, startTime)
-		trace.Step("Txn call finished", utiltrace.Field{Key: "err", Value: err})
 		if err != nil {
+			span.AddEvent("Txn call failed", attribute.String("err", err.Error()))
 			return err
 		}
-		trace.Step("Transaction committed")
+		span.AddEvent("Txn call succeeded")
```

Comment:
this phrasing right before checking `if !txnResp.Succeeded` is confusing...

Reply:
Probably `txn call completed` is better

Reply:
Changed to `txn call completed`